storage/pool/defaultpools.go

```diff
@@ -4,8 +4,6 @@
 package pool
```

Comment:
This whole file is deleted in master, pending work to fix provider registration.